.gitmodules

```diff
@@ -28,6 +28,3 @@
 	path = contracts/solidity-lib
 	url = https://github.com/Uniswap/solidity-lib
 	branch = v2.1.0
-[submodule "contracts/vault-v2"]
```

Comment:
We have some ideas for making submodule/contract management much more straightforward. Sorry if this is a pain point!